websites/K/Kinopoisk/presence.ts

```diff
@@ -0,0 +1,215 @@
+const presence = new Presence({
+	clientId: "1034799018980679680",
+});
+
+let video = {
+	duration: 0,
+	currentTime: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Где-то на сайте",
+			smallImageText: "Kinopoisk",
+		},
+		[privacy, time] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("time"),
+		]),
+		{ hostname, pathname } = document.location,
+		textContent = (tags: string) => {
+			return document.querySelector(tags)?.textContent?.trim();
+		},
+		pageTitle = (string: string) => {
+			return document
+				.querySelector<HTMLTitleElement>("title")
+				.textContent.split(string);
+		},
```

Comment:
Use functions instead of 'const functions'. ie functions invoked with the `function` keyword

Reply:
> Use functions instead of 'const functions'. ie functions invoked with the `function` keyword

Will be done.